be/src/service/starrocks_main.cpp

```diff
@@ -83,14 +80,17 @@ int main(int argc, char** argv) {
     if (argc > 1 && strcmp(argv[1], "meta_tool") == 0) {
         return meta_tool_main(argc - 1, argv + 1);
     }
-    // Check if print version or help.
+    bool as_cn = false;
```

Comment:
naming could be important to mental model: I guess what you want to emphasis there, is to start a node without an storage engine, so name like `with_storage_engine` could be better?

Reply:
Will rename it to `without_storage_engine`